pkg/apis/v1alpha5/machine.go

```diff
@@ -37,8 +37,8 @@ type MachineSpec struct {
 	// Kubelet are options passed to the kubelet when provisioning nodes
 	// +optional
 	Kubelet *KubeletConfiguration `json:"kubelet,omitempty"`
-	// MachineTemplateRef is a reference to an object that defines provider specific configuration
-	MachineTemplateRef *ProviderRef `json:"machineTemplateRef,omitempty"`
+	// ProviderRef is a reference to an object that defines provider specific configuration
+	ProviderRef *ProviderRef `json:"providerRef,omitempty"`
```

Comment:
Why did we swap this back? Just curious.

Reply:
Discussed with @njtran offline. I think we all agree this should be `MachineTemplateRef` going forward. Changing it back 👍